charts/nautobot/values.yaml

```diff
@@ -236,6 +235,23 @@ nautobot:
             value: 2
             periodSeconds: 15
 
+  # Add a Persistent Volume Claim for /static files of Nautobot
+  persistence:
+    # --- Enable PVC and the relevant volumes
+    enabled: false
+    # --- [Kubernetes StorageClass Name](https://kubernetes.io/docs/concepts/storage/storage-classes/), for the creation of Persistent Volume Claim
+    storageClass: ""
```

Comment:
Should we change this to `storageClassName` to be more explicit?

Reply:
Agreed, changed it to `storageClassName`